.github/workflows/update-docs-in-website.yml

```diff
@@ -0,0 +1,50 @@
+name: Update latest Glee documentation in the website
```

Comment:
the workflow and script looks good.

but did anyone try to first check manually if this is enough? Website docs will not just digest what you send that easily out of the box. I quickly looked into your `docs` dir and noticed:
- you have subfolders
- you have no front matters in markdown files, like title/weights in https://github.com/asyncapi/cli/blob/master/docs/context.md

My recommendation, try to do the same that you do here in the workflow, manually on a website, and see if your docs will render. I would suggest a prerequisite for this PR is to have a draft PR on website that showcases that after merging this PR it will work like a charm

Reply:
sure, I will be doing this part. Thanks

Reply:
@afzal442 Thanks! :+1: :rocket:

Reply:
> you have no front matters in markdown files

sorry @derberg I forgot to tell you about that is you could find those on `/docs/pages/*` folder.

Reply:
moreover, this [PR](https://github.com/asyncapi/glee/pull/538) is up to migrate those remaining pages to `/pages/` folder. Hope that makes sense now

Reply:
folks, you can now add `get-global-docs-autoupdate` topic to this repo, and ask @KhudaDad414 to run https://github.com/asyncapi/.github/actions/workflows/global-replicator.yml against this repo